src/classes/GE_GiftEntryController.cls

```diff
@@ -178,16 +175,14 @@ public with sharing class GE_GiftEntryController {
 
     /*******************************************************************************************************
     * @description Run the DataImport process on a single gift
-    * @param @param diRecord DataImport record to be processed
+    * @param diRecord DataImport record to be processed
     * @param isDryRun Boolean to set if the process should only return import results and not process
     * @param updateGift Boolean to determine our matching criteria.
     *                   true = "single match or create" and means we are updating
     *                   false = "do not match"
-    * @return void
     */
     private static void processDataImport(DataImport__c diRecord, Boolean isDryRun, Boolean updateGift) {
```

Comment:
@bdvorachek - would it be worth updating all these calls to match "dataImport" or should we only worry about the @AuraEnabled calls? Is that over-reach?

Reply:
fixed